### PR TITLE
chore(react-ui) try to build all when reactui is changed

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/apps/transfer/src/pages/transfer/account-transactions/index.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ const CheckTransactions: FC = () => {
             <StyledSmallField>
               <Select
                 leadingText={t('Chain')}
-                onChange={(e) => setChain(e.target.value)}
+                onChange={(e: any) => setChain(e.target.value)}
```
Comment (jessevanmuijden) [inline packages/apps/transfer/src/pages/transfer/account-transactions/index.tsx:134]: how about using
`onChange={(e: SyntheticEvent<HTMLSelectElement>) =>`

Comment (sstraatemans) [reply packages/apps/transfer/src/pages/transfer/account-transactions/index.tsx]: yeah that was already changed in other PR. I just wanted my build to succeed for a test :)
Thanks it should be good now

**Review by webpro — APPROVED** [review]
Feels a bit odd, but let's go

**Comment by webpro** [comment]
Closing this in favor of #687

**Pull request closed without merging**